moveit_ros/moveit_servo/tests/test_utils.cpp

```diff
@@ -153,7 +153,6 @@ TEST(ServoUtilsUnitTests, ApproachingSingularityScaling)
   const moveit::core::JointModelGroup* joint_model_group =
       robot_state->getJointModelGroup(servo_params.move_group_name);
 
-  rclcpp::sleep_for(std::chrono::milliseconds(500));
```

Comment:
@ibrahiminfinite Not sure if they are related to the problem but why do we need these sleeps in tests?